src/CustomerBundle/Block/ProfileMenuBlockService.php

```diff
@@ -58,7 +58,7 @@ public function configureSettings(OptionsResolver $resolver): void
         ]);
     }
 
-    private function getMenu(BlockContextInterface $blockContext): ItemInterface
+    protected function getMenu(BlockContextInterface $blockContext): ItemInterface
```

Comment:
Why is this needed?

Reply:
getMenu is override method from ```Sonata\BlockBundle\Block\Service\MenuBlockService```

I got error:
```In ProfileMenuBlockService.php line 82:
                                                                                                                                                                                             
   Compile Error: Access level to 
   Sonata\CustomerBundle\Block\ProfileMenuBlockService::getMenu() must be protected (as in 
   class Sonata\BlockBundle\Block\Service\MenuBlockService) or weaker
```
See your [comment](https://github.com/sonata-project/ecommerce/pull/634#discussion_r362586924). Sorry, i did't check it. Anyway it is not diffrence between protected and private method when class is final.

Reply:
Can you also retag ecommerce 3.2? It is big error. I didnt see it becouse sandbox cache correct version :(

Reply:
Please add a changelog note, I will create a patch version